barretenberg/cpp/src/barretenberg/ultra_honk/ultra_composer.hpp

```diff
@@ -17,46 +17,13 @@ template <IsUltraFlavor Flavor_> class UltraComposer_ {
     using CircuitBuilder = typename Flavor::CircuitBuilder;
     using ProvingKey = typename Flavor::ProvingKey;
     using VerificationKey = typename Flavor::VerificationKey;
-    using PCS = typename Flavor::PCS;
     using CommitmentKey = typename Flavor::CommitmentKey;
     using VerifierCommitmentKey = typename Flavor::VerifierCommitmentKey;
     using ProverInstance = ProverInstance_<Flavor>;
     using VerifierInstance = VerifierInstance_<Flavor>;
-    using FF = typename Flavor::FF;
     using Transcript = typename Flavor::Transcript;
-    using CRSFactory = srs::factories::CrsFactory<typename Flavor::Curve>;
```

Comment:
Core changes: all this data goes away; compute_commitment_key functionality removed; also there's no reason to explicitly set special member functions to `default` unless the compiler requires it (I think that can happen but that isn't typically required?).